pandas/core/describe.py

```diff
@@ -175,10 +175,13 @@ def describe(self, percentiles: Sequence[float]) -> DataFrame:
             ldesc.append(describe_func(series, percentiles))
 
         col_names = reorder_columns(ldesc)
-        d = concat(
-            [x.reindex(col_names, copy=False) for x in ldesc],
-            axis=1,
-            sort=False,
+        d = cast(
+            DataFrame,
+            concat(
+                [x.reindex(col_names, copy=False) for x in ldesc],
```

Comment:
`x.reindex(col_names, copy=False)` was previously revealed to be of type `Any`. Now that it's typed, the result of `concat` is `Frame | Series`, so the cast is necessary

Reply:
presumably at some point we'll overload concat s.t. axis=1 returns DataFrame, at which point this cast will be unnecessary?

Reply:
yeah, exactly!